tests/scripts/test_create_superuser.py

```diff
@@ -1,3 +1,8 @@
+import sys
+
+# This is a hacky workaround to test the scripts in a subdir
+sys.path.insert(0, "scripts")
```

Comment:
If we add an `__init__.py` file to the `scripts` directory then they could be imported without needing to patch the path. Then:

```py
from scripts.create_superuser import (
    collect_username_and_password,
    create_user_and_client,
)
```

Reply:
i tried everything I could to avoid adding this hack ☹️ figured its fine for now, especially since its just for tests